configure.ac

```
@@ -428,6 +428,10 @@ fi
 AM_CONDITIONAL(BUILD_DOCUMENTATION, test x$enable_documentation = xyes)
 
 GOBJECT_INTROSPECTION_CHECK([1.40.0])
+if test "x$found_introspection" = "xyes" -a "x$enable_asan" = xyes; then
+   AC_MSG_WARN([Instrospection was found succesfully but will be deactivated since it can't be used when sanitizing flags are on.])
+   AC_MSG_WARN([use \"--enable-asan=no\" to ensure introspection can be enabled])
+fi
```

```suggestion
AS_IF([test "x$found_introspection" = "xyes" && test "x$enable_asan" = xyes], [
   AC_MSG_WARN([Instrospection was found succesfully but will be deactivated since it can't be used when sanitizing flags are on.])
   AC_MSG_WARN([use \"--enable-asan=no\" to ensure introspection can be enabled])
])
```

(`test x -a y` is not portable to all shells, and `AS_IF` is preferred over `if`/`fi` for annoying reasons involving the way Autoconf's macro language works.)